static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -232,6 +257,98 @@ const errorHelpTypeResourceMap: Record<
       linksByPlatform: {},
     },
   },
+  [ErrorHelpType.NEXTJS_ROUTER_NOT_MOUNTED]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Next.js applications when the useRouter hook is used incorrectly. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>NextRouter not mounted errors</b>}
+      ),
+      links: [
+        {
+          text: t('Fixing "NextRouter was not mounted" errors in Next.js'),
+          link: 'https://sentry.io/answers/error-nextrouter-was-not-mounted/',
+        },
+      ],
+      linksByPlatform: {},
+    },
+  },
+  [ErrorHelpType.UNBOUND_LOCAL_ERROR]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Python applications when a variable is defined in both global and local contexts. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>Unbound local errors</b>}
+      ),
+      links: [
+        {
+          text: t('Fixing "unbound local" errors in Python'),
+          link: 'https://sentry.io/answers/unbound-local-error/',
+        },
+      ],
+      linksByPlatform: {},
+    },
+  },
+  [ErrorHelpType.NODEJS_CANNOT_FIND_MODULE]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Node.js applications when an imported module cannot be found. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>Cannot find module errors</b>}
+      ),
+      links: [
+        {
+          text: t('Fixing "cannot find module" errors in Node.js'),
+          link: 'http://sentry.io/answers/how-do-i-resolve-cannot-find-module-error-using-node-js/',
+        },
+      ],
+      linksByPlatform: {},
+    },
+  },
+  [ErrorHelpType.NO_MODULE_NAMED]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Python applications when a module that does not exist is imported. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>Module not found errors</b>}
```

Comment:
seems like the error is more of an import error not ModuleNotFound, can we change it to match the Sentry Answer